priv/repo/migrations/20180816191344_create_scenes.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Ret.Repo.Migrations.CreateScenes do
+  use Ecto.Migration
+
+  def change do
+    create table(:scenes, prefix: "ret0", primary_key: false) do
+      add :scene_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true
```

Comment:
nit: Not sure what convention we're using for parentheses around function arguments. They have parens in most other parts of the code. I just copied the existing migrations.

Reply:
ah we're using the elixir reformatter (new in last release) -- you can rig up ale in vim to use it